### DISCUSSION TITLE
plot Field at a specific frequency?

### DISCUSSION THREAD

**Discussion by berzoll** [NanoComp/meep#2826] [Q&A]
is there a way to plot and get the animation (with Animate2D) of a field filtered at a specific frequency (or filtered in a range of frequencies) ? 

thanks

**Comment by stevengj** [comment]
You can use the DFT field feature to get the field at any desired frequency.  I don't think we have any built-in plotting to animate this field $\times \exp(-i\omega t)$, but it would be nice to have.

You can certainly do this yourself — just take the DFT field for a given frequency $f$, and plot it multiplied by $\exp(-i2\pi f t)$ for a range of $t$ values $\in [0, 1/f)$.  Then loop the animation of these plots
